modules/text/include/opencv2/text/erfilter.hpp

```diff
@@ -227,6 +227,11 @@ features: hole area ratio, convex hull ratio, and number of outer inflexion poin
 CV_EXPORTS_W Ptr<ERFilter> createERFilterNM2(const Ptr<ERFilter::Callback>& cb,
                                                   float minProbability = (float)0.3);
 
+CV_EXPORTS_W  Ptr<ERFilter> createERFilterNM1(const String& filename,
```

Comment:
Could you please guide me where I need to provide documentation? If I just add comments would it be sufficient?